### ISSUE TITLE
learning rate for 1024

### ISSUE THREAD

**Issue by happybunny56** [Yuanshi9815/OminiControl#51]
Which learning rate do you use to train model on 1024 resolution? 
Thanks!

**Comment by Yuanshi9815** [comment]
Hi @happybunny56!

Merry Christmas! 🎄

We use the [Prodigy optimizer](https://github.com/konstmish/prodigy) which doesn't require manually setting a learning rate.
